Makefile

```
@@ -41,7 +41,7 @@ endif
 		plugin_id_gpu="${PLUGIN_ID}-gpu-cuda${CUDA_VERSION}"; \
 		echo "Modifying a few files to make the plugin GPU-ready. Fasten your seatbelt."; \
 		sed -i "" "s/${PLUGIN_ID}/$${plugin_id_gpu}/g" plugin.json; \
-		sed -i "" "s/]/,\"GPU\"]/g" plugin.json; \
+		sed -i "" "s/: \[/: \[\"GPU\",/g" plugin.json; \
 		sed -i "" "s/\"label\": \"Forecast\"/\"label\": \"Forecast (GPU - CUDA ${CUDA_VERSION})\"/g" plugin.json; \
```

We also need the `(deprecated)` label in the plugin name
```suggestion
		sed -i "" "s/\"label\": \"Forecast (deprecated)\"/\"label\": \"Forecast (GPU - CUDA ${CUDA_VERSION} - deprecated)\"/g" plugin.json; \
```